ktypes/preempt-rt/preempt-rt.cfg

```diff
@@ -16,3 +131,950 @@ CONFIG_NET_SCH_ETF=y
+CONFIG_ROMFS_FS=m
+CONFIG_QUOTA=n
+CONFIG_AUTOFS_FS=y
+
+#
+# CD-ROM/DVD Filesystems
+#
+
+#
+# DOS/FAT/NT Filesystems
+#
+CONFIG_FAT_FS=m
+
+#
+# Pseudo filesystems
+#
+CONFIG_CONFIGFS_FS=m
+
+#
+# Layered filesystems
+#
+
+# temporary workaround for unionfs
+
+#
+# Miscellaneous filesystems
+#
+
+#
+# Network File Systems
+#
+CONFIG_CIFS=m CONFIG_CIFS_XATTR=y
+CONFIG_CIFS_POSIX=y
+CONFIG_CODA_FS=n
+CONFIG_AFS_FS=n
+
+#
+# Partition Types
+#
+
+#
+# Native Language Support
+#
+CONFIG_NLS_CODEPAGE_437=m
+CONFIG_NLS_CODEPAGE_737=m
+CONFIG_NLS_CODEPAGE_775=m
+CONFIG_NLS_CODEPAGE_850=m
+CONFIG_NLS_CODEPAGE_852=m
+CONFIG_NLS_CODEPAGE_855=m
+CONFIG_NLS_CODEPAGE_857=m
+CONFIG_NLS_CODEPAGE_860=m
+CONFIG_NLS_CODEPAGE_861=m
+CONFIG_NLS_CODEPAGE_862=m
+CONFIG_NLS_CODEPAGE_863=m
+CONFIG_NLS_CODEPAGE_864=m
+CONFIG_NLS_CODEPAGE_865=m
+CONFIG_NLS_CODEPAGE_866=m
+CONFIG_NLS_CODEPAGE_869=m
+CONFIG_NLS_CODEPAGE_936=m
+CONFIG_NLS_CODEPAGE_950=m
+CONFIG_NLS_CODEPAGE_932=m
+CONFIG_NLS_CODEPAGE_949=m
+CONFIG_NLS_CODEPAGE_874=m
+CONFIG_NLS_ISO8859_8=m
+CONFIG_NLS_CODEPAGE_1250=m
+CONFIG_NLS_CODEPAGE_1251=m
+CONFIG_NLS_ASCII=m
+CONFIG_NLS_ISO8859_1=m
+CONFIG_NLS_ISO8859_2=m
+CONFIG_NLS_ISO8859_3=m
+CONFIG_NLS_ISO8859_4=m
+CONFIG_NLS_ISO8859_5=m
+CONFIG_NLS_ISO8859_6=m
+CONFIG_NLS_ISO8859_7=m
+CONFIG_NLS_ISO8859_9=m
+CONFIG_NLS_ISO8859_13=m
+CONFIG_NLS_ISO8859_14=m
+CONFIG_NLS_ISO8859_15=m
+CONFIG_NLS_KOI8_R=m
+CONFIG_NLS_KOI8_U=m
+CONFIG_NLS_UTF8=m
+
+#
+# Distributed Lock Manager
+#
+
+#
+# Instrumentation Support
+#
+
+#
+# Probes
+#
+
+#
+# Event types
+#
+
+#
+# Kernel hacking
+#
+
+#
+# Security options
+#
+
+#
+# PaX
+#
+
+#
+# PaX Control
+#
+
+#
+# Non-executable pages
+#
+
+#
+# Address Space Layout Randomization
+#
+
+#
+# Miscellaneous hardening features
+#
+
+#
+# Grsecurity
+#
+
+#
+# Address Space Protection
+#
+
+#
+# Role Based Access Control Options
+#
+
+#
+# Filesystem Protections
+#
+
+#
+# Kernel Auditing
+#
+
+#
+# Executable Protections
+#
+
+#
+# Network Protections
+#
+
+#
+# Sysctl support
+#
+
+#
+# Logging Options
+#
+CONFIG_SECURITY=y
+CONFIG_SECURITY_NETWORK=n
+
+#
+# Cryptographic options
+#
+CONFIG_CRYPTO_HMAC=y
+CONFIG_CRYPTO_NULL=y
+CONFIG_CRYPTO_MD4=y
+CONFIG_CRYPTO_MD5=y
+CONFIG_CRYPTO_SHA1=y
+CONFIG_CRYPTO_SHA256=y
+CONFIG_CRYPTO_SHA512=y
+CONFIG_CRYPTO_WP512=m
+CONFIG_CRYPTO_DES=y
+CONFIG_CRYPTO_BLOWFISH=m
+CONFIG_CRYPTO_TWOFISH=m
+CONFIG_CRYPTO_SERPENT=m
+CONFIG_CRYPTO_AES=y
+CONFIG_CRYPTO_CAST5=m
+CONFIG_CRYPTO_CAST6=m
+CONFIG_CRYPTO_TEA=m
+CONFIG_CRYPTO_ARC4=m
+CONFIG_CRYPTO_KHAZAD=m
+CONFIG_CRYPTO_ANUBIS=m
+CONFIG_CRYPTO_DEFLATE=y
+CONFIG_CRYPTO_MICHAEL_MIC=m
+CONFIG_CRYPTO_CRC32C=y
+CONFIG_CRYPTO_TEST=m
+
+#
+# Layered filesystem encryption
+#
+CONFIG_FS_ENCRYPTION=y
+CONFIG_ENCRYPTED_KEYS=y
+CONFIG_CRYPTO_SEQIV=y
+CONFIG_CRYPTO_CTR=y
+
+#
+# Hardware crypto devices
+#
+
+#
+# Library routines
+#
+CONFIG_LIBCRC32C=y
+CONFIG_ZLIB_DEFLATE=y
+>>>>>>> 32d6b478 (config: replace CONFIG_AUTOFS4_FS with CONFIG_AUTOFS_FS)
```

Comment:
There is a conflict here, and it also adds a bunch of non related changes.

Reply:
Done